sonic_platform_base/sonic_storage/ssd.py

```diff
@@ -49,10 +81,18 @@ def __init__(self, diskdev):
             "StorFly"  : { "utility" : VIRTIUM,  "parser" : self.parse_virtium_info },
             "Virtium"  : { "utility" : VIRTIUM,  "parser" : self.parse_virtium_info },
             "Swissbit" : { "utility" : SMARTCTL, "parser" : self.parse_swissbit_info },
+            "Micron"   : { "utility" : SMARTCTL, "parser" : self.parse_micron_info },
+            "Intel"    : { "utility" : SMARTCTL, "parser" : self.parse_intel_info },
             "Transcend" : { "utility" : TRANSCEND, "parser" : self.parse_transcend_info },
         }
 
         self.dev = diskdev
+        self._fetch_parse_info(diskdev)
+
+        StorageCommon.__init__(self, diskdev)
+
+    def _fetch_parse_info(self, diskdev):
```

Comment:
I would suggest to remove underscore in the start of the API. Underscore convention is used only for private APIs. Since this is public and will be called from the daemon, it would be misleading

Reply:
Fixed this in latest commit